src/main/java/org/gitlab4j/api/AbstractApi.java

```diff
@@ -88,6 +88,31 @@ protected String urlEncode(String s) throws GitLabApiException {
             throw new GitLabApiException(e);
         }
     }
+    
+    /**
+     * Encode a string to be used as in-path argument for a gitlab api request.
+     * 
+     * Standard URL encoding changes spaces to plus signs, but for arguments that are part of the path,
+     * like the :file_path in a "Get raw file" request, gitlab expects spaces to be encoded with %20.
+     * 
+     * @param s the string to encode
+     * @return encoded version of s with spaces encoded as %2F
+     * @throws GitLabApiException if encoding throws an exception
+     */
+    protected String urlEncodeForPath(String s) throws GitLabApiException {
+        try {
```

Comment:
Why not just add `encoded = encoded.replace("+", "%20");` to `urlEncode(String s)`

I'm pretty sure  `urlEncode(String s)` not processing `+` is an oversight